scenes/MainScene.tscn

```diff
@@ -1,8 +1,9 @@
-[gd_scene load_steps=6 format=2]
+[gd_scene load_steps=7 format=2]
```

Comment:
Steps = 7....? Is this the number of resources you are loading in the scene?

Reply:
No idea! Anything in a .tscn file is generated by the IDE, haha